src/types.ts

```diff
@@ -0,0 +1,16 @@
+export interface CarouselItem {
+  id: string;
+  name: string;
+  description: string;
+  cookTime: string;
+  image: string;
+  details: string;
+  nutrient: string;
+  favoriteIcon: string;
+  frontButtonText: string;
+  backButtonText: string;
+  ingredients: string[];
+  instructions: string[];
+}
```

Comment:
Yes! I had used the following for the Recipes component:
`  type Attributes = { 
    name: string, 
    time_to_cook: number, 
    nutrient: string, 
    image: string,
    description: string,
    ingredients: string[],
    instructions: string[]
  }

  type Recipe = {
    id: number, 
    type: string,
    attributes: Attributes
  }`
  This is based on the data from Postman